modules/setting/setting.go

```diff
@@ -720,6 +722,8 @@ func NewContext() {
 		SSH.StartBuiltinServer = false
 	}
 
+	SSH.BuiltinServerUser = sec.Key("BUILTIN_SSH_USER").MustString("")
```

Comment:
Can this be changed to `SSH.BuiltinServerUser = sec.Key("BUILTIN_SSH_USER").MustString(RunUser)`?

Reply:
It is set to the runuser if blank a bit further down. I cannot combine them because this one requires the server sec variable, and the runuser is set further down.

Reply:
Yes, I saw that line. Can you move this setup line down as `BuiltinSSHServerUser` is not used between lines 725 and 922?